projects/kit/components/range/range.style.less

```diff
@@ -56,17 +58,25 @@
     &::after {
         .fullsize(absolute, inset);
 
+        --t-direction: right;
```

Comment:
We already have this:
https://github.com/taiga-family/taiga-ui/blob/main/projects/core/components/root/root.style.less#L22

Reply:
https://github.com/taiga-family/taiga-ui/pull/11474/commits/81e4d55589804103ee964a91ee1e726e21aa8ae9